.github/workflows/formulaCI.yml

```diff
@@ -0,0 +1,78 @@
+name: Formula CI
+# this workflow will
+# - run formula validation step - test/validate-formula.sh
+# - deliver the package content to the configured repository
+# - submit the new package content to the upstream repository
+on: [push, pull_request]
+env:
+  PACKAGE_NAME: saphanabootstrap-formula 
+jobs:
+
+  validation:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Add salt repository
+      uses: myci-actions/add-deb-repo@4
+      with:
+        repo: deb http://repo.saltstack.com/apt/ubuntu/18.04/amd64/latest/ bionic main
```

Comment:
Please, don't use `latest`. Latest might bring some not tested elements. In openSUSE we ship `salt v3000`, so let's use this